config/clients/js/template/example/example1/example1.mjs

```diff
@@ -0,0 +1,286 @@
+import { Credentials, CredentialsMethod, FgaApiValidationError, OpenFgaClient, TypeName } from "@openfga/sdk";
+
+let credentials;
+if (process.env.FGA_CLIENT_ID) {
+  credentials = new Credentials({
+    method: CredentialsMethod.ClientCredentials,
+    config: {
+      clientId: process.env.FGA_CLIENT_ID,
+      clientSecret: process.env.FGA_CLIENT_SECRET,
+      apiAudience: process.env.FGA_API_AUDIENCE,
+      apiTokenIssuer: process.env.FGA_API_TOKEN_ISSUER
+    }
+  });
+}
+
+const fgaClient = new OpenFgaClient({
+  apiUrl: process.env.FGA_API_URL || "http://localhost:8080",
+  storeId: process.env.FGA_STORE_ID, // not needed when calling `createStore` or `listStores`
+  authorizationModelId: process.env.FGA_AUTHORIZATION_MODEL_ID, // optional, recommended for production,
+  credentials
+});
+
+console.log("Listing stores");
+try {
+  const stores = await fgaClient.listStores();
+  console.log(`Stores count: ${stores.stores.length}`);
+} catch (error) {
+  console.error(`error: ${error}`);
+  process.exit(1);
```

Comment:
Instead of having process.exit everywhere, maybe we can put these in a main function and at the end have:

```
main().catch(err => {
  console.error(`err:`, err);
  process.exit(1);
})
```

Reply:
Yeah that makes more sense, will refactor